src/Microsoft.AspNet.Mvc.Razor/RazorServiceCollectionExtensions.cs

```diff
@@ -24,5 +25,25 @@ public static class ServiceCollectionExtensions
         {
             services.Configure(setupAction);
         }
+
+        /// <summary>
+        /// Adds an initialization callback for a given <see cref="ITagHelper"/> type.
+        /// </summary>
+        /// <remarks>
+        /// The callback will be invoked on the <see cref="ITagHelper"/> of the specified type before the
+        /// <see cref="ITagHelper.ProcessAsync(TagHelperContext, TagHelperOutput)"/> method is called.
+        /// </remarks>
+        /// <typeparam name="TTagHelper">The type of <see cref="ITagHelper"/> being initialized.</typeparam>
+        /// <param name="services">The services available in the application.</param>
+        /// <param name="initialize">An action to initialize the <see cref="ITagHelper"/>.</param>
+        public static void InitializeTagHelper<TTagHelper>(
```

Comment:
Does this need to be fluent? The `WithControllersAsServices` in MvcServiceCollectionExtensions is